evap/evaluation/tools.py

```diff
@@ -112,15 +97,18 @@ def sort_formset(request: HttpRequest, formset: BaseFormSet) -> None:
         formset.forms.sort(key=lambda f: f.cleaned_data.get("order", 9001))
 
 
-def date_to_datetime(date: datetime.date) -> datetime.datetime:
-    return datetime.datetime(year=date.year, month=date.month, day=date.day)
-
-
 def vote_end_datetime(vote_end_date: datetime.date) -> datetime.datetime:
     # The evaluation actually ends at EVALUATION_END_OFFSET_HOURS:00 of the day AFTER self.vote_end_date.
     return date_to_datetime(vote_end_date) + datetime.timedelta(hours=24 + settings.EVALUATION_END_OFFSET_HOURS)
 
 
+def translate(**kwargs):
```

Comment:
Was this intentionally moved? If not, let's move it back :)

Reply:
oops. Fixed.